### DISCUSSION TITLE
User friendly platform that people can use to report اسامی-مزدوران-جمهوری-اسلامی

### DISCUSSION THREAD

**Discussion by bbgunuuuu** [WeAreMahsaAmini/Core#47] [Ideas]
We need a user-friendly platform that people can annoyingly or with just an email login report and vote and comment on to list the Islamic Republic Mercenary in and outside Iran. 
We particularly need one to find all of them and in Canada and report to the Canadian government. 
Any Ideas?

**Comment by IranFreedom** [comment]
I like it as they should be afraid.

**Reply by bbgunuuuu** [reply]
Do you have any suggestion as what platform or software to use? 
We want it to be user-friendly and have the option to add picture, and links.

**Reply by Ahmad-Raza18** [reply]
Allah Hoo (Allah Hoo) is a traditional Sufi Zikr consisting of the word for ALLAH (Arabic: الله) run together three times, followed by Truth (haqq):
https://youtu.be/3Zfh5wH1bYQ

**Comment by mrnobud** [comment]
Great idea! 
Not just canada though we will need it everywhere 

Building the app is easy, but how would you ensure the information is real? 
Do we want anyone to submit a name? Or do we need some sort of proof?

**Reply by bbgunuuuu** [reply]
no proof needed; it can be multi comment like this page plus vote system for people to comment under comment and also vote up or down.  The accusation is enough, its for the future or current governments to prosecute. More proof is better.

**Reply by ghost** [reply]
Then how do you prevent them from accusing us?

**Reply by ghost** [reply]
^^^ This is my question too. We need a vetting process to make sure the opposition (Or any other political movement) can't hijack the system.
The only system that I can think of in which something similar is applied, is a blockchain POW technology. Unfortunately, I don't have in-depth knowledge of how a system like that can be implemented. Any thoughts?

**Reply by HakhaAndish** [reply]
definitely we need some sort of proof here if we want it to be effective,  prone to hijack and reliable for future reference, otherwise it'll be a superficial project. I suggest to invite lawyers in our socials to join. Also, it's against moral if we suspect someone and later on find out it was false (yet another reason on why we need proof even before accusation).

**Reply by Setareh4IRAN** [reply]
check this link : https://instagram.com/attorneyrazaghi?igshid=YmMyMTA2M2Y=
he is a lawyer working on this topic, he also put an email for people who want to help him :
https://www.instagram.com/p/CjGAlEwr4Th/?igshid=YmMyMTA2M2Y=

Let's make a connection and have the max effect.

**Comment by Sam-mnz** [comment]
Hi 👋 

I have been working on this for a while, and I have faced a roadblock, I have heard that there could be a defamation sue against us if we do this without enough proofs, anyone can guide us through this?

**Reply by bbgunuuuu** [reply]
no one owns this , its public and people can have freedom of speech and accuse or claim.

**Reply by ghost** [reply]
This is Canadian law: Defamation refers to harming another person’s reputation by making a false written or oral statement about that person to a third party ... There are a number of legal defenses against defamation: ... You can claim “responsible communication on matters of public importance,” which allows journalists to report false allegations if the news is urgent and of public importance, and if the journalist made an effort to verify the information. Even if the statement is false, the public has an interest in this type of discussion being legally permissible.

Disclaimer: I am not a lawyer, definitely talk to a lawyer first.

**Reply by Setareh4IRAN** [reply]
check this link : https://instagram.com/attorneyrazaghi?igshid=YmMyMTA2M2Y=
he is a lawyer working on this topic, he also put an email for people who want to help him :
https://www.instagram.com/p/CjGAlEwr4Th/?igshid=YmMyMTA2M2Y=

Let's make a connection and have the max effect.

**Reply by BardyTimeSpace** [reply]
I think the focus should be on identifying  for those working for the government inside Iran at this stage.  Once the revolution comes to fruition, the new government will be able to create a taskforce to identify and request extradition for those who operate or taken public funds outside the country.  In today's digital world, tracking those should not be an issue.
imho Identifying and organizing the information on those who are killing, torturing and contributing to the oppression of  people inside Iran which would includes city, organization (basij, sepah, ettelaat, naja, ...)  and basic information, can be a good tool to help the movement push forward.

**Comment by ghost** [comment]
I also have a similar idea. Think of the same CelebrityWall but full of faces of people actively supporting the Islamic Republic (or encouraging to ignore our movement). The difference is that they are not always Celebrities. 

The main features of such a platform can be the following:

1. Ability to check the tweet/post/video/blog/etc. in which they have showed their support for the Islamic Republic. Or even when stating something against the MahsaAmini movement. **Goal**: When they find out that they are recorded in out platform, they might thing of deleting their tweet/post/video/blog/etc. Besides, they are revealed and then people start reporting them to their governments (in case they live outside of Iran).
2. Since we need a way to be sure the reported cases are truly supporting the government, there should be a number of admins to check the validity and give a thumbs up (like 3 or 5 admins). The better idea is to make this administration decentralised but I think it would be difficult at this point of time.
3. Ability to give a +1 by viewers to check the most wanted ones! 

Note: not all the features are essential, but we can start with the first one and see how far it goes.

**Reply by spdkh** [reply]
I have been thinking about something to recognize faces of the guards attacking people on the streets or plain cloths. I have seen some projects like that using face recognition but I dont know who is working on such projects.

**Comment by bbgunuuuu** [comment]
Khamenei is dead. The beast of the corruption network is massive and has
many heads and different forms. They are stealing the wealth of the public.
Please hurry. We need to start simple and add features. What is important
is that it must be very easy to use for any person. let's give the power to
people to choose. numbers and vote will show. What we need to do is to
prevent duplication, so check if same name is already there. another
problem is that sometimes the name is unknown but the footage or picture
are there. I thinks its better to start simple and fix and improve over
time. Country is very important because we need to report many here in
canada to Canadian government for sanction. Please hurry. any idea how we
can start quick?

**Reply by kaveh6** [reply]
Whether or not "Khamenei is dead" is irrelevant to the cause and simply does not matter . I agree with the rest of the sentence

**Reply by ghost** [reply]
I agree that we need to start quick, and add features after.
Is there any "Reddit" like open-source platform we can start with?
I think a platform with the ability of vote-ups and vote-down, and with the ability to share a link like "reddit" or "hackernews" is best to start with.

**Reply by kaveh6** [reply]
https://github.com/WeAreMahsaAmini/Core/discussions/107

**Comment by ghost** [comment]
Here is an example of one them on Linkedin (living in Canada): https://www.linkedin.com/feed/update/urn:li:activity:6979759114566410240/
![image](https://user-images.githubusercontent.com/84380122/192742582-c8044198-2eb6-4df0-8d95-4d8a86de6fa1.png)

Maybe connected to the gov, maybe not. But they should know that they are being watched, and so either they remove their posts, or we remove it by reporting.

**Reply by ghost** [reply]
What about the people who liked this post? Are they also connected? I think this is turning into a witch hunt. I believe "innocent until proven guilty" is a good approach, there should be strong evidence.

**Reply by HakhaAndish** [reply]
There is no such a post "anymore". But good to know and archive them!!

**Comment by bbgunuuuu** [comment]
Redit can’t add pictures in comments, average user might not be able to
find a way to make image link.

**Reply by ghost** [reply]
Image can be fabricated so it's not that much important (at least for the start). The most important thing is to be able to share the link to the source.

**Comment by bbgunuuuu** [comment]
Use case:
I have picture, name and information about someone, no link. How I report?

**Comment by ksaffarian** [comment]
Who can claim someone is a "mozdoor" for sure? I mean, due to ambigious nature of such claims, it can potentially be something dangerous that can impact innocent people.
So, there should be a fair review and voting over each nomination.
Of course everyone know all the famous "mozdoor"!

So, it is a tricky thing to implement, IMO.

**Reply by kaveh6** [reply]
Identifying a mozdoor is relatively easy as they leave such obvious trails and footprints . but having an automated way of doing this could easily be misused in the wrong hands .

**Reply by HakhaAndish** [reply]
we need "peer review" group including good number of people to vote the provided proof. Good amount of people just for having more certainty in the vote, like 100 people (which we already have).

**Reply by Setareh4IRAN** [reply]
check this link : https://instagram.com/attorneyrazaghi?igshid=YmMyMTA2M2Y=
he is a lawyer working on this topic, he also put an email for people who want to help him :
https://www.instagram.com/p/CjGAlEwr4Th/?igshid=YmMyMTA2M2Y=

Let's make a connection and have the max effect.

**Comment by ghost** [comment]
To make it simpler, and also easier to implement:

**Forget about recognising a "Mozdoor".**

**Focus on reporting a act of support of the regime.**

**Let's make reporting such cases the main goal. And have the platform as a place to find/have a record of them.**

**Reply by ziXet** [reply]
Agree. We should pick a good domain name for this.

**Comment by ziXet** [comment]
How about using IPFS?
https://docs.ipfs.tech/how-to/websites-on-ipfs/single-page-website/#install-ipfs-desktop

**Comment by bbgunuuuu** [comment]
Thanks, we are building platform for expression. You go in-force it.

**Comment by seraheazari** [comment]
سلام به همگی دوستان. ما یک پلتفرم برای اینکار درست کردیم. لطفا برای عضو شدن به این [لینک](https://mozdoorpedia.tribeplatform.com) مراجعه کنید.
در صورتی که تمایل ندارید اطلاعات شخصی شما نمایان بشه٫ لطفا از اسم مستعار و ایمیل متفاوتی از ایمیل شخصی خودتون استفاده کنید.

**Reply by Setareh4IRAN** [reply]
check this link : https://instagram.com/attorneyrazaghi?igshid=YmMyMTA2M2Y=
he is a lawyer working on this topic, he also put an email for people who want to help him :
https://www.instagram.com/p/CjGAlEwr4Th/?igshid=YmMyMTA2M2Y=

Let's make a connection and have the max effect.

**Reply by kaveh6** [reply]
سورس این پروژه ی شما کجاس ؟

**Comment by bbgunuuuu** [comment]
خدا پدر و‌مادرت را بیاموزه ! دستت در نکنه شیر زن .

**Comment by bbgunuuuu** [comment]
There is an error when one wants to login using Google

**Reply by seraheazari** [reply]
تست کردیم و ثبت نام با گوگل کار می کنه. امکانش هست اسکرین شات بذارید؟

**Comment by ZahraEconomist** [comment]
به نظرم رفتن به سراغ کامنت توییتر و لایک لینکدین برای شناسایی مزدور مخالف آزادیه اظهارنظره. اگر قراره  لیستی تهیه بشه باید با مسٔولین رد بالای مملکتی٬ افراد خانواده اشون  و لیست  اموال شخصیشون شروع بشه. و عکس آدمهایی که تو خیابون باتوم به دست گرفتن یا با افتخار تو شرکتهای سانسور اینترنت کار میکنن.

**Reply by Setareh4IRAN** [reply]
check this link : https://instagram.com/attorneyrazaghi?igshid=YmMyMTA2M2Y=
he is a lawyer working on this topic, he also put an email for people who want to help him :
https://www.instagram.com/p/CjGAlEwr4Th/?igshid=YmMyMTA2M2Y=

Let's make a connection and have the max effect.

**Reply by kaveh6** [reply]
@Setareh4IRAN are you a spammer ? why do you keep posting this link ?

**Comment by Gavazneasli** [comment]
Hey folks,
As a UX/UI Designer, I designed this prototype:
https://www.figma.com/proto/kpWsWhf7Rl9i3Peu1txWK7/Mozdooryab-V0.1?node-id=2%3A2&scaling=scale-down-width&page-id=0%3A1&starting-point-node-id=2%3A2&hide-ui=1
Let me know what you think when you have a chance to check it out.
I definitely can improve it.

**Reply by ziXet** [reply]
Good job!

**Reply by bbgunuuuu** [reply]
being user friendly is very important? how are we implementing this?

**Comment by AnonymousMaximus2** [comment]
چند نکته خیلی مهم:

حساب کنید در طول 40 سال روزانه 5 میلیون بشکه نفت تولید شده جمعا میشود 73 میلیارد بشکه نفت، میانگین نفت یک زمانی دوره احمدی نژاد به 150 دلار رسیده. و زمانی به زیر 100 دلار رسیده و الان 110 دلار است میانگین فرض کنیم 100 دلار، میشود 7.3 تریلیون دلار،

در طول 40 سال به غیر از نفت، دولت ایران شرکتهای دولتی، بانکها، پتروشیمی ها و معادن طلا و معادن آهن و .... همه را استخراج کرده و فروخته، فقط سالیانه 50 میلیارد دلار پتروشیمی تولید میکنه، که توی 40 سال میشه 2 تریلیون دلار، و اگر فرض کنیم 700 میلیارد دلار صادرات آهن و معادن و طلا و جواهرات قیمتی و سنگهای معدنی مانند مرمر و درآمدهای ارزی بانک مرکزی باشد، جمعا در طول 40 سال 10 تریلیون دلار ثروت مردم ایران را به جیب زده

به غیر از این 10 تریلیون دلار ذخایر نفت و گاز و معادن که زیر زمین هست حدود 150 تریلیون دلار است.

یک نکته مهم اینکه این پولها توی بانکهای بزرگ دنیا بصورت ناشناس در گردش هستند،

هر پولی در هر بانکی باشد میتوان تا 10 برابر کردیت و اعتبار و وام گرفت.

یعنی این مافیای بین المللی دولت ایران اعتبار و کردیت و وامی حدود 100 تریلیون دلار دارد


عده ای ناشناس توی دنیا هستند که این 10 تریلیون دلار ثروت مردم ایران را دارند دست به دست میکنند

هر چند وقت یک بار یه جائی بلوکه میشه و بعد از چند سال آزاد میشه

توی اخبار چند سال پیش زمان امضای قرارداد برجام اعلام شد توی ترکیه آمریکا 100 میلیارد دلار بلوکه شده را آزاد کرده.

یا چند سال پیش 9 میلیارد دلار توی هند بلوکه شده بود.

یا توی کره جنوبی چندین میلیارد دلار بلوکه شده


عدد یک تریلیون دلار آنقدر بزرگ است که توی عقل و مغز ما جا نمیشود.

اگر این عدد را بین 80 میلیون ایرانی تقسیم کنید با 100 تریلیون دلار کردیت میشود برای هر ایرانی وام ارزی بین المللی معادل 1.25 میلیون دلار معادل 40 میلیارد تومن وام گرفت که باهاش برای خودش بیزنس راه اندای کنه یا حدود 4 میلیارد تومن پول نقد بهش داد بره برای خودش راحت زندگی کنه اگر یک خانواده 5 نفره باشند جمعا به یک خانواده 20 میلیارد تومن نقد و یا 200 میلیارد تومن وام و کردیت برای راه انداززی بیزنس تعلق میگیره و این همه ثروت را حرام زاده های جمهوری اسلامی دارند توی سح دنیا با قدرتهای دنیا شطرنج های سیاسی بای میکنند

هیچ وقت اختلاسهای بزرگ را نمیگذارند مردم اصلا فکرش را بکنند.

حتی افرادی مانند بابک زنجانی و محمود خاوری توی کانادا و شهرام جزایری هم با اختلاسهای میلیارد دلاری شان درمقابل حقایق پشت پرده اعداد خورد هستند.

اقا زاده ها هر کدام چند میلیون دلار داشته باشند هزار نفر آقا زاده جمعشان میشود چند میلیارد دلار، معادل اختلاس خاوری،
و کل اختلاس بابک زنجانی هم چند میلیارد دلار بود. شهرام جزایری هم که عددی نبود جوجه بود،

دکل نفتی که گم شد قیمتش در حدود 100 میلیون دلار بود.


تبلیغات بین المللی کرده اند که خانواده یهودی روچیلد ثروتشان 2 تریلیون دلار است.

ولی کافر های جمهوری اسلامی که هیچ بوئی از اسلام و انسانیت نبرده اند، توی دست شان 10 تریلیون دلار پول نقد و 100 تریلیون دلار کردیت بانکی دارند، و مردم داخل ایران از گرسنگی میمیرند.

هر چند گاهی از سود این ثروت عظیم مبلغ کم چند صد میلیون دلار به حزب الله لبنان یا یمن یا سوریه یا سایر کشورها کمک میکنند

به چین و روسیه باج میدهند و حتی پول خرید هواپیماهای بدون سرنشین روسیه را میدهند و در جنگهای دنیا اسپانسور میشوند.

دنبال اینها باشید، پشت اینها آقازاده ها و فرندان مسئولین نیستند،

بگردید ببینید پشت اینها چه کسانی هستند

و البته هر کسی پشت این چیزها باشه نمیگذارند زنده بماند و هر جای دنیا باشد ترورش میکنند

این پولها برای بازی قمار و شطرنج با کشورهای روسیه و چین در مقابل آمریکا و اروپا است.

و با کسی هم شوخی ندارند

من بیشتر از اینها نمیتوانم حرف بزنم سر نخ را دادم خودتان دنبالش بروید اگر جراتش را دارید

من جرات بیشتر از این حرف زدن را ندارم.

**Comment by Kimiabr** [comment]
برای یه همچین کاری باید حواستان را جمع کنید. 
کسی با توطئه چینی از میان افراد به دنبال حذف شخصی به دلایل متفرقه نباشد.
اسناد و مدارک کاملا قابل استناد برای صحبت ها و ایرادات بر آن ها وجود داشته باشد.
با وکلای متخصص در این خصوص و مربوط به هر کشور مربوطه صحبت شود تا اهانت و اعاده ی حیثیت پیش نیاید.
اطمینان از منبع اطلاعات و نیز اطمینان از حمایت بدون قید و شرط حکومت، در غیر این صورت بی گناهی را بدون مدرک و اثبات و بر اثر نظرات شخصی دچار مشکل کرده اید.
آزادی بیان اشخاص به هیچ عنوان نباید زیر سوال برده شود و در صورتی که به دلیلی آزادی بیان نقض شود تمام اقدامات این جنبش زیر سوال میرود و از منظر قانونی پیگیری میشود.
موضوعاتی که در این افراد مورد بررسی قرار میگیرند باید تاپیک ها و اقداماتِ غیرقانونی آن ها باشد، نه صرفا اخلاقیات و نظرات خلاف عقاید گروه دیگر از مردم.
بهتر است کمی بیشتر و عمیق تر به این پروژه نگاه کرد و بعد شروع به انجام فعالیت نمود.

**Comment by AnonymousMaximus2** [comment]
چند نکته اشاره کنم

افرادی که اقا زاده هستند و پول دارند نمی آیند که بگویند بله حق با شماست اشتباه کردم

آنها زرنگ تر از شما هستند. و تمامی هفت خط بازی ها را قبلا رفته و پولهایی به دست آورده اند.

اگر یک ذره اشتباه بکنید به راحتی میتوانند چند ده هزار دلار پول بدهند و یک وکیل اروپائی و یا آمریکائی بگیرند و یک پرونده اعاده حیثیت defamation علیه شما در دادگاه تشکیل بدهند و نه تنها شما کاری نمیتوانید بکنید بلکه کل زندگی شما را مصادره کرده و میتوانند شما را توی زندانهای آمریکا و اروپا بیاندازند.

اگر هم با ادمی کله گنده و مهم سرو کار داشته باشید که آنها اصلا اهل دادگاه و کیل بازی نیستند به راحتی پولهای میلیون دلاری میدهند تا هر جای دنیا باشید مامور ترور استخدام میکنند.

من خودم توی یکی از کشورها جایی خیلی راحت علنا در حضور 10 نفر آدم یکی از نوچه هایشان تهدیدم کرد گفت خرج کشتن تو فقط 2 هزار دلاره  

تازه ادمهای اصلی نبودند،  یک نوچه زیر دستی بود.

و من دیگه از انجا رفتم ...

پس حساب شده و سنجیده حرکت کنید تا برایتان دردسر درست نشود.

**Comment by Setareh4IRAN** [comment]
check this link : https://instagram.com/attorneyrazaghi?igshid=YmMyMTA2M2Y=
he is a lawyer working on this topic, he also put an email for people who want to help him : 
https://www.instagram.com/p/CjGAlEwr4Th/?igshid=YmMyMTA2M2Y=

Let's make a connection and have the max effect.

**Comment by matinirani** [comment]
Any update or help on this needed?

**Comment by matinirani** [comment]
من فک می کنم باید چندین کتگوری در نظر گرفت:
1) افرادی که مردم رو میزنند = عکس آن ها موجود است و دیتاش پابلیک منتشر میشه و درخواست اطلاعات شخصی میشه از بقیه.
2) افرادی که در پشت صحنه مزدور هستن. ثابت کردن اینکه آن ها واقعا مزدور هستند یا نه کار سختیه و نیاز به کار قانونی داره. ولی میشه یه لیست ازشون داشت ولی دیتا جایی منتشر نشه تا وقتی که بشه دادگاهی برگزار کرد.

**Comment by bbgunuuuu** [comment]
Can you elaborate?

**Comment by bbgunuuuu** [comment]
We need a platform so the crowd list their :

Name , picture, location, accusations and proof / comment.  Let’s not help
them hide.

**Reply by BardyTimeSpace** [reply]
If the platform is open to the crowd to upload, update and modify, it'll also be open to sabotage. For that reason, the data may need to be collected, screened and uploaded by an admin (can add tags such as confirmed, pending verification,...).  This will be a source of information for freedom fighters in Iran and it'll serve as a tool for post-revolution legal proceedings again regime collaborators. I'd recommend an interactive map of the country where you can click into cities > district > organization > IR collaborator info as you mentioned

**Comment by babooloo** [comment]
These exists: 
1. [IranCourt](https://irancourt.com/)
2. [MozdoorYab](https://mozdooryab.com/)
